devtools/conda/dev/meta.yaml

```diff
@@ -21,7 +21,6 @@ requirements:
     - sphinx_rtd_theme ==0.1.9
 
   run:
-    - ambermini
```

Comment:
Why are you removing ambermini? This will cause conda installs to fail if users don't have amber installed.